### ISSUE TITLE
Doc?

### ISSUE THREAD

**Issue by MaBeuLux88** [FaustXVI/gol-rs#1]
J'arrive pas à le lancer.
Il n'y a pas de doc ;-) ! 
0/20

**Comment by FaustXVI** [comment]
Effectivement, un minimum de connaissance rust son requises : https://doc.rust-lang.org/cargo/commands/cargo-run.html
